public/app/features/alerting/unified/components/rule-editor/alert-rule-form/simplifiedRouting/AlertManagerRouting.tsx

```diff
@@ -1,29 +1,33 @@
 import { css } from '@emotion/css';
-import React from 'react';
+import React, { useState } from 'react';
 
 import { GrafanaTheme2 } from '@grafana/data';
 import { Alert, CollapsableSection, Icon, Link, LoadingPlaceholder, Stack, Text, useStyles2 } from '@grafana/ui';
 import { AlertManagerDataSource } from 'app/features/alerting/unified/utils/datasource';
 import { createUrl } from 'app/features/alerting/unified/utils/url';
 
 import { useContactPointsWithStatus } from '../../../contact-points/useContactPoints';
+import { ContactPointWithMetadata } from '../../../contact-points/utils';
 
 import { ContactPointDetails } from './contactPoint/ContactPointDetails';
 import { ContactPointSelector } from './contactPoint/ContactPointSelector';
 import { MuteTimingFields } from './route-settings/MuteTimingFields';
 import { RoutingSettings } from './route-settings/RouteSettings';
 
+interface RouteSettings {
+  muteTimeIntervals: string[];
+  overrideGrouping: boolean;
+  groupBy: string[];
+  overrideTimings: boolean;
+  groupWaitValue: string;
+  groupIntervalValue: string;
+  repeatIntervalValue: string;
+}
 interface AlertManagerManualRoutingProps {
   alertManagerContactPoint: {
```

Comment:
Nice job splitting this up! Do we still need the extra `alertManagerContactPoint` key here? This looks like an object with just a single key 🤔

Reply:
💯

Reply:
done!